cmd/run.go

```diff
@@ -122,20 +121,8 @@ func start(cliCtx *cli.Context) error {
 		log.Fatal(err)
 	}
 
-	st := newState(cliCtx.Context, c, l2ChainID, []state.ForkIDInterval{}, stateSqlDB, eventLog, needsExecutor, needsStateTree)
+	st, currentForkID := newState(cliCtx.Context, c, tmpEthMan, l2ChainID, stateSqlDB, eventLog, needsExecutor, needsStateTree, false)
 
-	etherman, err := newEtherman(*c, st)
```

Comment:
Is this safe to remove?

Reply:
> Is this safe to remove?

`etherman` instance has not been removed.
It is now being created at line 130

Reply:
I see now. Thanks.